### DISCUSSION TITLE
Official Docker image

### DISCUSSION THREAD

**Discussion by kamiradi** [google-deepmind/mujoco#1039] [General]
Hi,

I was wondering if there was an official docker image that ships the latest version of the library? Please correct me if I'm wrong, but there seems to be a few earlier versions of the library on standard images on Docker hub, but not anything recent.

**Comment by saran-t** [comment]
We don't have official Docker images not any plan for these. However, note that the binaries that we ship (via GitHub releases) have no external dependency so if anyone wants to, they can just containerise by downloading the tarball and extracting it.
